packages/app-desktop/gui/ErrorBoundary.tsx

```diff
@@ -30,6 +32,19 @@ interface Props {
 	children: any;
 }
 
+interface BannerProps {
+	isVisible: boolean;
+}
+
+const SwitchToNewEditorBanner = (props: BannerProps) => <BannerContent
+	acceptMessage={_('Switch to the new editor')}
+	onAccept={() => Setting.setValue('editor.legacyMarkdown', false)}
+	visible={props.isVisible}
+>
+	{_('The legacy Markdown editor might be incompatible with a number of plugins. We recommend using the new editor.')}
```

Comment:
The legacy editor appears to have crashed due to an incompatibility with a plugin. We recommend using the new editor
```suggestion
	{_('The legacy Markdown editor appears to have crashed due to an incompatibility with a plugin. We recommend using the new editor.')}
```